src/Common/tests/System/Net/Prerequisites/Servers/CoreFxNetCloudService/WebServer/WebServer.csproj

```diff
@@ -206,4 +206,6 @@
   <Target Name="AfterBuild">
   </Target>
   -->
+  <Target Name="AnnotateProjectsWithConfiguration" />
```

Comment:
Yeah, I added this here because this project doesn't include the dir.targets where those targets are defined, and given it is under a **\tests\**\*.csproj path it is being considered as a test project that is required to build.